dev/workflow/applications/gefs.py

```diff
@@ -167,7 +166,9 @@ def get_task_names(self):
             tasks += ['gempak']
 
         if options['nens'] > 0:
-            tasks += ['atmos_ensstat']
+            tasks += ['efcs', 'atmos_ensstat']
```

Comment:
`efcs` should be defined above, before the product tasks.  `atmos_ensstat` and `awips` should be here, after the products.